.github/workflows/danger.yml

```diff
@@ -1,5 +1,5 @@
 name: Danger
-on: [pull_request]
+on: [ pull_request ]
```

Comment:
Waarom is dit nu met spaties?

Reply:
iets met webstorm, heb het terug gedraaid en uitgezet